mmdet3d/datasets/pipelines/formating.py

```diff
@@ -264,3 +265,112 @@ def __repr__(self):
         repr_str += f'(class_names={self.class_names}, '
         repr_str += f'with_gt={self.with_gt}, with_label={self.with_label})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class GetBEVDetInputs():
+    """Generate the inputs of BEVDet including the images and the
+    transformation information for the Lift-Splat-Shoot view transformer."""
+
+    def get_rot(self, rad):
+        """Generate 2D rotation matrix according to the input radian.
+
+        Args:
+            rad (float): Ratation magnitude in radian.
+
+        Returns:
+            torch.Tensor: The 2D rotation matrix in shape of (2, 2).
+        """
+        return torch.Tensor([
+            [np.cos(rad), np.sin(rad)],
+            [-np.sin(rad), np.cos(rad)],
+        ])
+
+    def get_post_transform(self, resize, crop, flip, rotate):
+        """Generate 3D translation and rotation matrix according to the image
+        view data transformation.
+
+        Args:
+            resize (float): Scale of resize.
+            crop (tuple(int)): Range of cropping in format of (lower_w,
+                lower_h, upper_w, upper_h).
+            flip (bool): Flag of flip operation.
+            rotate (float): Magnitude of rotation in angle.
+
+        Returns:
+            tuple(torch.Tensor): The 3D translation and rotation matrix.
```

Comment:
Please describe the returns rightly.

Reply:
@JingweiZhang12 I found so many formats for tuple returns in this repo. Which is exactly the recommended one? 
''' How about this?
        Returns:
            Tuple[torch.Tensor]:
                post_rot_3d: Rotation matrix of image view data transforation.
                post_tran_3d: Translation matrix of image view data 
                    transforation.
'''

Reply:
It's OK as long as the description is clear and has no ambiguity. Sometimes better to descript the shape if necessary.
''' 
  Returns:
     Tuple[torch.Tensor, torch.Tensor]:
        - post_rot_3d: Rotation matrix of image view data transformation.
        - post_tran_3d: Translation matrix of image view data transformation.
'''
The docstring extension in VSCode may be helpful.